scripts/contractInteraction/amm.py

```diff
@@ -345,6 +346,28 @@ def withdrawFromRBTCWrapperProxy(tokenAddress, to, amount):
     wrapperProxy = Contract.from_abi("RBTCWrapperProxy", address=conf.contracts['RBTCWrapperProxy'], abi=abi, owner=conf.acct)
     wrapperProxy.withdraw(tokenAddress, to, amount)
 
+def transferOwnershipAMMContractsToGovernance(contractAddress, destinationAddress, contractName=''):
```

Comment:
```suggestion
def transferOwnershipAMMContractsToGovernance(contractAddress, newOwnerAddress, contractName=''):
```

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/450/commits/94459ccdbffb4c8e2fa42981a355178c076b9a2f)